external/nyc311/stat.py

```diff
@@ -67,7 +71,7 @@ def get_311_statistics(
             total_complaints_query_zip = len(complaints_query_zip_df)
             # complaints of the desired type in the zip code of interest that were closed
             closed_complaints_query_zip = len(
-                dict(tuple(complaints_query_zip_df.groupby("status")))["Closed"]
+                dict(tuple(complaints_query_zip_df.groupby("status"))).get("Closed", [])
```

Comment:
interesting, I assume this returns an empty array if no attribute with "closed" key is found? nifty

Reply:
Correct